test/test_aistore.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import random
+import string
+import tempfile
+import unittest
+
+from torchdata.datapipes.iter import AISFileLister, AISFileLoader
+
+try:
+    from aistore.client.api import Client
+    from aistore.client.errors import AISError, ErrBckNotFound
+
+    AIS_CLUSTER_ENDPT = "http://localhost:8080"
+
+    HAS_AIS = Client(AIS_CLUSTER_ENDPT).is_aistore_running()
+except ImportError or ConnectionError:
```

Comment:
Noob question because I am not familiar with this syntax. Should it be
`except (ImportError, ConnectionError):`?

Reply:
fixed